### DISCUSSION TITLE
Copy to clipboard results of embedded code blocks

### DISCUSSION THREAD

**Discussion by aikruger** [brianpetro/obsidian-smart-connections#409] [Ideas]
Would it be possible to add a copy to clipboard functionality to the embedded code block so that the results of the smart-search can be pasted into a separate document as it is presented in search results?

**Comment by brianpetro** [comment]
Hey @aikruger 

    ```smart-connections
    test query
    ```

Using the above in a note should render a smart view based on the embedding for "test query".

Is that what you're looking for? Or something else?

Thanks
🌴

**Reply by aikruger** [reply]
Looks good, if it copies the results of the query rather than the query itself

**Comment by LubosKolouch** [comment]
Yes, the ability to copy the results is critical. What good is to see the result, when you can't copy them and work with them further?
